File: 001-100/003/largestPrimeFactor.py
from math import sqrt, ceil

def isPrime(n):
    for i in range(2, ceil(sqrt(int(n)))):
        if(int(n) % i == 0):
            return False
    return True

def largestPrime(n):
    largestPrime = 1
    for i in range(1, int(n)):
        if(isPrime(i) and i > largestPrime):
            largestPrime = i
    return largestPrime

if __name__ == "__main__":
    n = input("Enter an integer n: ")
    print("\nLargest Prime below {:,}".format(int(n)))
    print("{:,}".format(largestPrime(n)))

File: 001-100/003/largestPrimeFactor.php
<?php
  // Check if a number is prime
  function isPrime($n){
    for ($i = 2; $i < $n; $i++) {
      if($n % $i == 0){ return False; }
    }
    return True;
  }
  // Find the largest prime number below n
  function largestPrime($n){
    $largestPrime = 1;
    for ($i = 1; $i < $n ; $i++) {
      if (isPrime($i) && $i > $largestPrime && $n % $i == 0){ $largestPrime = $i; }
    }
    return $largestPrime;
  }

  // Get user input
  $prompt = "Enter an integer n: ";
  $n = readline($prompt);
  $largestPrime = largestPrime($n);
  // Print results
  echo "\nLargest Prime below $n";
  echo "\n$largestPrime\n"
?>
